src/plot/xy-plot.js

```diff
@@ -413,7 +413,8 @@ class XYPlot extends React.Component {
       marginBottom,
       marginRight,
       innerHeight,
-      innerWidth
+      innerWidth,
+      pixelRatio: window.devicePixelRatio
```

Comment:
might be good to generalize this for node env testing, eg make a default at the top that is something like

```javascript
const DEFAULT_PIXEL_RATIO = window && window.devicePixelRatio || 1;
```

also, would you mind alphabetizing this list?

Reply:
Sounds like a good idea. I've also alphabetized the list.